my-app/src/App.js

```diff
@@ -1,10 +1,25 @@
-import './App.css';
-import Header from './components/Header';
-
+import "./App.css";
+import { useEffect, useState } from "react";
+import Header from "./components/Header";
+import ContentList from "./components/ContentList";
+require("dotenv").config();
 function App() {
+  const [content, setContent] = useState();
+  useEffect(() => {
+    const apikey = process.env.REACT_APP_UNSPLASH_KEY;
+    fetch(
+      `https://newsapi.org/v2/top-headlines?sources=bbc-news&apiKey=${apikey}`
+    )
+      .then((data) => data.json())
+      .then((res) => setContent(res.articles));
+  }, []);
```

Comment:
I have one last suggestion here. try to keep your `App component` clean as much as possible. If there is some business logic/ functionalities that can be placed somewhere else without side-effects then it should be welcomed. I'd recommend moving this `useEffect` into a separate file, `src/hooks/useFetch.jsx` for instance.
you need to move your `content` state with it as well and return it from there.
```js
const useFetch = (url) => { 
.....
useEffect(() => {
      fetch(url)
        .then((data) => data.json())
        .then((res) => {
          setContent(res.articles));
          setIsLoading(false);
          })
        .catch((err) => {
          setIsLoading(false);
          setError(err);
        })
  }, []);
return { content, isLoading, error.... }
}
```